test/helpers/views/declaration/SavedDeclarationsMessages.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package helpers.views.declaration
+
+trait SavedDeclarationsMessages {
+
+  private val prefix: String = "saved.declarations"
+
+  val title: String = prefix + ".title"
```

Comment:
I think it'll be better to not create val prefix, but e.g. title will have whole message key. The same with other messages. It'll be easier if you need to find what message is inside, so easier is to highlight messageKey and search for it

Reply:
I agree.  I was just following the pattern set by the other tests.   I'll change it and we can change the other ones sometime